yarn.lock

```diff
@@ -1,5707 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Please **do not** `git rm yarn.lock` :-)
My previous comment was just about committing changes if the package.json is untouched.